renderdoc/common/globalconfig.h

```diff
@@ -36,8 +36,9 @@
 
 /////////////////////////////////////////////////
 // Build/machine configuration
-#if defined(__LP64__) || defined(_WIN64) || defined(__x86_64__) || defined(_M_X64) || \
-    defined(__ia64) || defined(_M_IA64) || defined(__aarch64__) || defined(__powerpc64__)
+#if defined(__LP64__) || defined(_WIN64) || defined(__x86_64__) || defined(_M_X64) ||        \
+    defined(__ia64) || defined(_M_IA64) || defined(__aarch64__) || defined(__powerpc64__) || \
+    defined(__riscv64) && __riscv_xlen == 64
```

Comment:
Can you please add parentheses around this group? Even though operator precedence means it's unnecessary, it's easier to read and clearer of the intention.